.github/workflows/databases-nightly.yml

```diff
@@ -0,0 +1,53 @@
+name: databases-nightly
+
+on:
+  pull_request: # TODO: Remove before merge
```

Comment:
```suggestion
```

Should be deleted before merge.

Reply:
Of course I forgot this 😂 

Removed now

Reply:
Thanks! I just wanted to create a PR, because I also just thought I probably forgot to remove that.